s2i-liberty/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# This image provides a base for building and running WildFly applications.
+# It builds using maven and runs the resulting artifacts on WildFly 10.1.0 Final
+
+FROM websphere-liberty:webProfile7
```

Comment:
the builder image can be any. The runtime image is created by IBM based on debian. I changed the user to our standard 1001. I think we want to keep the IBM's provided image as our base image because it increases the chances for our customers that IBM will support it.